docs/bundles.md

```diff
@@ -0,0 +1,353 @@
+      pulling oci://ghcr.io/stefanprodan/modules/redis:7.0.10
+      using module timoni.sh/redis version 7.0.10
+      installing redis in namespace podinfo
+      Namespace/podinfo created
+      applying master
+      ServiceAccount/podinfo/redis created
+      ConfigMap/podinfo/redis created
+      Service/podinfo/redis created
+      Deployment/podinfo/redis-master created
+      PersistentVolumeClaim/podinfo/redis-master created
+      waiting for 5 resource(s) to become ready...
+      resources are ready
+      applying replica
+      Service/podinfo/redis-readonly created
+      Deployment/podinfo/redis-replica created
+      waiting for 2 resource(s) to become ready...
+      resources are ready
+      applying instance podinfo
+      pulling oci://ghcr.io/stefanprodan/modules/podinfo:6.3.5
+      using module timoni.sh/podinfo version 6.3.5
+      installing podinfo in namespace podinfo
+      ServiceAccount/podinfo/podinfo created
+      Service/podinfo/podinfo created
+      Deployment/podinfo/podinfo created
+      waiting for 3 resource(s) to become ready...
+      resources are ready
+      ```
+
+List the instances in the `podinfo` namespace:
+
+=== "command"
+
+      ```sh
+      timoni list -n podinfo
+      ```
+
+=== "output"
+
+     ```text
+     NAME    MODULE                                          VERSION LAST APPLIED         
+     podinfo oci://ghcr.io/stefanprodan/modules/podinfo      6.3.5   2023-04-10T16:20:07Z    
+     redis   oci://ghcr.io/stefanprodan/modules/redis        7.0.10  2023-04-10T16:20:00Z
+     ```
+
+List the instance resources and their rollout status:
+
+=== "command"
+
+      ```sh
+      timoni status redis -n podinfo
+      ```
+
+=== "output"
+
+     ```text
+     ServiceAccount/podinfo/redis Current Resource is current
+     ConfigMap/podinfo/redis Current Resource is always ready
+     Service/podinfo/redis Current Service is ready
+     Service/podinfo/redis-readonly Current Service is ready
+     Deployment/podinfo/redis-master Current Deployment is available. Replicas: 1
+     Deployment/podinfo/redis-replica Current Deployment is available. Replicas: 1
+     PersistentVolumeClaim/podinfo/redis-master Current PVC is Bound
+     ```
+
+See an instance module reference and its digest:
+
+=== "command"
+
+      ```sh
+      timoni inspect module redis -n podinfo
+      ```
+
+=== "output"
+
+     ```text
+     name: timoni.sh/redis
+     repository: oci://ghcr.io/stefanprodan/modules/redis
+     version: 7.0.10
+     digest: sha256:7fcb6f6918902c0dedc2ba4545fbdeca119a04644a53400af15b977e3921c600
+     ```
+
+## Writing a Bundle spec
+
+A Bundle file must contain a definition that matches the following schema:
+
+```cue
+#Bundle: {
+	apiVersion: string
+	instances: [string]: {
+		module: {
+			url:     string
+			digest?: string
+			version: *"latest" | string
+		}
+		namespace: string
+		values: {...}
+	}
+}
+```
+
+Bundle files can contain arithmetic operations,
```

Comment:
Think it may be worth to link to a couple of CUE things being referred to (e.g. https://cuelang.org/docs/tutorials/tour/expressions/operators/, https://cuelang.org/docs/tutorials/tour/expressions/interpolation/), to help people coming from Helm.